### DISCUSSION TITLE
premailer-rails

### DISCUSSION THREAD

**Discussion by manuelmeurer** [bootstrap-email/bootstrap-email#97] [Help] [answered]
Hey, I just discovered bootstrap-email, it looks great!

Quick question: until now, I used the `premailer-rails` gem.
Is it advised/necessary to keep it in my Gemfile?
bootstrap-ruby has a dependency on `premailer` as well, so`premailer-rails` is probably not needed?

**Comment by stuyam** [comment, marked as answer]
Correct, bootstrap email will pull in the required dependencies automatically and you do not need to specify them in your gemfile if you are using the alpha of the 1.0.0 👍

**Reply by manuelmeurer** [reply]
Thanks! 🙇‍♂️
